pkg/featflag/features.go

```diff
@@ -89,7 +89,10 @@ var (
 )
 
 func init() {
-	c, _ := config.New()
+	c, err := config.New()
```

Comment:
`// If the user does not have a config.toml we still want them to be able to use the environment based feature flag variables`